saw-remote-api/src/SAWServer.hs

```diff
@@ -59,27 +60,44 @@ import SAWServer.Exceptions
 
 type SAWCont = (SAWEnv, SAWTask)
 
+type CryptolAST = P.Expr P.PName
+
 data SAWTask
   = ProofScriptTask
-  | LLVMCrucibleSetup ServerName [SetupStep]
+  | LLVMCrucibleSetup ServerName [SetupStep LLVM.Type]
+  | JVMSetup ServerName [SetupStep JavaType]
 
 instance Show SAWTask where
   show ProofScriptTask = "ProofScript"
   show (LLVMCrucibleSetup n steps) = "(LLVMCrucibleSetup" ++ show n ++ " " ++ show steps ++ ")"
+  show (JVMSetup n steps) = "(JVMCrucibleSetup" ++ show n ++ " " ++ show steps ++ ")"
```

Comment:
The constructor doesn't include `Crucible`, but the string representation does. These should agree, right?

Reply:
Oh, yes, they should.